app/components/provider_interface/filter_component.html.erb

```diff
@@ -36,7 +36,9 @@
         </div>
       <% end %>
 
-      <div class="moj-filter__options">
+      <!-- Div made focusable to 'catch' focus from checkboxes. see https://github.com/DFE-Digital/apply-for-teacher-training/pull/2640
```

Comment:
No need for this comment here or in the CSS — GitHub's merge commit will include the PR reference & we'll be able to get back to it from the diff. Possibly it ought to go into the commit message.

Reply:
@duncanjbrown normally I'd agree - but in this case I think what I'm doing is unexpected and it might be helpful to note this is intentionally weird.

Reply:
As you like!